state/src/check.rs

```diff
@@ -536,21 +491,14 @@ fn check_verifreg_against_miners(
     for claim in verifreg_summary.claims.values() {
         // all claims are indexed by valid providers
         let maddr = Address::new_id(claim.provider);
-        let miner_summary = match miner_summaries.get(&maddr) {
+        let _miner_summary = match miner_summaries.get(&maddr) {
             None => {
                 acc.add(format!("claim provider {} is not found in miner summaries", maddr));
                 continue;
             }
             Some(summary) => summary,
         };
 
-        // all claims are linked to a valid sector number
-        acc.require(
-            miner_summary.sectors_with_deals.get(&claim.sector).is_some(),
-            format!(
-                "claim sector number {} not recorded as a sector with deals for miner {}",
-                claim.sector, maddr
-            ),
-        );
+        // XXX all claims are linked to a valid sector number
```

Comment:
Please replace the XXX with a link to an issue, or just remove it and add a TODO item to #1342